tests/playwright/src/model/pages/main-page.ts

```diff
@@ -94,4 +99,33 @@ export abstract class MainPage extends BasePage {
     }
     return undefined;
   }
+
+  async getRowsFromTableByStatus(status: string): Promise<Array<Locator>> {
+    await waitUntil(async () => await this.rowsAreVisible(), { sendError: false });
+
+    const table = this.content.getByRole('table');
+    if (table) {
+      const rows = await table.getByRole('row').all();
+      rows.shift(); //first row is the headers row
```

Comment:
You should check that rows.length is greater than 1 before shifting, since you need a minimum of length 2 in order to know there is an actual volume there, since first row is the header.